src/fediblockhole/__init__.py

```diff
@@ -238,6 +242,18 @@ def merge_blocklists(blocklists: list[Blocklist], mergeplan: str='max',
                 block = apply_mergeplan(block, newblock, mergeplan)
             merged.blocks[block.domain] = block
 
+        if len(save_block_audit_file) > 0:
```

Comment:
Yeah, I think this check should be `if save_block_audit_file:` which will match on None or empty list/string.

Could more explicitly require `if save_block_audit_file is not None:` but a wider capture of non-useful settings (like empty strings) here probably guards more against accidentally opening a wrong logfile, and matching the user intent.

I think I'd find the first method easier to interpret after a while away from the code. It took me a moment to figure out why this needed to be a positive-length item, and it's more that it needs to be something that can be opened as a file. Let's be kind to future us.